lib/components/atoms/Video/Video.story.js

```diff
@@ -0,0 +1,38 @@
+import React from 'react';
+import { storiesOf } from '@storybook/react';
+import defaultConfig from './Video.mock';
+
+// Import Styled Component to showcase variations
+import Video, { VideoVanilla } from '.';
+import Track, { TrackVanilla } from '../Track/Track';
+
+storiesOf('Atoms', module)
+  .addParameters({ jest: ['Video', 'VideoVanilla'] })
+  .add('Video Knobs', () => (
+    <VideoVanilla {...defaultConfig}>
+      {defaultConfig.trackdata.map(caption => (
+        <TrackVanilla
+          src={caption.src}
+          default={caption.isDefault}
+          kind="subtitles"
+          srcLang={caption.srcLang}
+          label={caption.label}
+          key={caption.id}
+        />
+      ))}
+    </VideoVanilla>
+  ))
+  .add('Video', () => (
+    <Video {...defaultConfig}>
```

Comment:
Your component can be used in 2 diff ways at least make one sample diff

Reply:
done

Reply:
I don't think so, what I meant was, there are 2 different ways people can use your component. The story book should document that. 

Maybe for VideoVanilla you can let the default Track render instead of passing a list

Reply:
fixed